src/Compiler/Interactive/fsi.fs

```diff
@@ -4618,7 +4617,7 @@ type FsiEvaluationSession
                 unresolvedReferences,
                 fsiOptions.DependencyProvider
             )
-            |> NodeCode.RunImmediateWithoutCancellation
+            |> Async.RunSynchronously
```

Comment:
I can't recall why I changed this to RunSynchronously, probably when fighting with fsi deadlocks.